recipes/ipart/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Since you are building from the github release, the license file is already in the tarball. If you switch to using pypi in the future, you will need to manually include the license file or make sure it is in the source distribution, which is currently is not.